site/content/kapp/docs/latest/config.md

```diff
@@ -80,6 +82,9 @@ For examples of rebase rules in use, see [HPA and Deployment rebase](hpa-deploym
      - `[existing, new]` – If field value is present in the `existing` resource on cluster, use that value, otherwise use the value in the `new` user input.
      - `[existing]` – Only look for field values in resources already on cluster, corresponding value you provide in new resource will be overwritten.
   - `resourceMatchers` (array) specifies rules to find matching resources. See various resource matchers below.
+  - `ytt` specifies choice as ytt for rebase rule. 
+  - `overlayContractV1` specifies ytt overlay feature used to patch YAML files. 
+  - `overlay.yml` YAMl document that is the overlay, describing the modification. 
```

Comment:
these keys should be indented, so that one can see that they are children

Reply:
Done